src/ios/CDVWKInAppBrowser.m

```diff
@@ -1175,6 +1175,18 @@ - (UIColor *)colorFromHexString:(NSString *)hexString {
 
 #pragma mark WKNavigationDelegate
 
+- (void)webView:(WKWebView *)theWebView didReceiveAuthenticationChallenge:(nonnull NSURLAuthenticationChallenge *)challenge completionHandler:(nonnull void (^)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable))completionHandler {
+    NSString *domain = theWebView.URL.absoluteURL.host;
+    NSDictionary *credentials = _browserOptions.basicauth[domain];
+    if([challenge.protectionSpace.authenticationMethod isEqualToString:NSURLAuthenticationMethodHTTPBasic] == YES && credentials) {
+        NSLog(@"didReceiveAuthenticationChallenge");
+        NSURLCredential *credential = [NSURLCredential credentialWithUser:credentials[@"user"] password:credentials[@"pass"] persistence:NSURLCredentialPersistenceForSession];
+        completionHandler(NSURLSessionAuthChallengeUseCredential, credential);
```

Comment:
Nice find on this callback and implementation looks great.  If you squint past the objective-c syntax, it looks remarkably similar to the android callback method. :)